<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScene</class>
 <widget class="QWidget" name="RegisterScene">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QLineEdit" name="regUsername">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regPassword">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register2Btn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="regConfirm">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip2label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>regUsername</zorder>
  <zorder>regPassword</zorder>
  <zorder>regConfirm</zorder>
  <zorder>tip2label</zorder>
  <zorder>register2Btn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
